src/routes/pois/components/PoiListItem.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+import * as React from 'react'
+import ListItem from '../../../modules/common/components/ListItem'
+import PoiModel from '../../../modules/endpoint/models/PoiModel'
+
+type PropsType = {|
+  poi: PoiModel
+|}
+
+class PoiListItem extends React.Component<PropsType> {
+  render () {
+    const {poi} = this.props
+    return (
+      <ListItem key={poi.path}
+                thumbnail={poi.thumbnail}
+                title={poi.title}
+                path={poi.path}>
+        <div>{poi.location.location}</div>
```

Comment:
I think we should also add the excerpt here, as we do for the events. TBD in #feature-poi

Reply:
I don't know. Probably most pois are just autogenerated event locations which doesn't have an excerpt. We could make it optional of course. @maxammann what do you think?

Reply:
excerpt and description are optional for events. @svenseeberg is this correct?

Reply:
I'll just leave it for now, we can add it later if an excerpt is provided.